neuralg/io/download_models.py

```diff
@@ -0,0 +1,27 @@
+from loguru import logger
+import requests
+import zipfile
+import io
+import os
+
+
+def download_models():
+    """Downloads and unzips the folder of model state dicts attached to the latest GitHub release.
+    Will save the folder as saved_models in the models subdirectory.
+    """
+    logger.info("Downloading models from the latest release")
+    # This is currently pre-release version
+    zip_url = "https://github.com/esa/neuralg/releases/download/v.0.0.1-alpha/saved_models.zip"
+    # This should be enabled for real release. Remember to tag release with latest.
+    # "https://github.com/esa/neuralg/releases/latest/download/saved_models.zip"
+    r = requests.get(zip_url)
+
+    logger.info("Download complete. Unzipping content")
+    z = zipfile.ZipFile(io.BytesIO(r.content))
+    saved_models_path = os.path.realpath(
+        os.path.join(
+            os.path.dirname(__file__),
+            "../models/",
+        )
+    )
+    z.extractall(saved_models_path)
```

Comment:
will this create the folder if it doesn't exist? just asking because these are the kind of things which work on one OS and not on the other.

Is there a test pypi I can try to test this?

Reply:
Aha, good point!  yes it is doing so on my OS, there is an almost up to date test pypi here:
 ```ssh 

python -m pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple/ neuralg

```
Created this before this PR though so a few quirks could be there

Reply:
Now there should be a testPyPi version with the lates good from this PyPi!